src/redux/actions/profileActions.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+import config from "../../config";
+import * as actionTypes from "../action_types";
+import history from "../../routes/history";
+
+const profileURL = `${config.BASE_URL}/user`;
+const token = localStorage.getItem("token");
+
+// profile loading
+export const setProfileLoading = () => ({
+  type: actionTypes.PROFILE_LOADING
```

Comment:
Why not import the specific action-type you need and use it here, why import all of them? As for the comments, please enquire from the rest of your teammates on how action creator comments are written and structured. No need for comments on thunks.

Reply:
@kimotho-njoki I have made the changes as requested.